Configurations/10-main.conf

```diff
@@ -456,9 +454,6 @@ sub vms_info {
     },
 
     # More attempts at unified 10.X and 11.X targets for HP C compiler.
-    #
-    # Chris Ruemmler <ruemmler@cup.hp.com>
-    # Kevin Steves <ks@hp.se>
```

Comment:
It's inconsistent. In some cases name and address is removed, in others just address. Then there are cases when it's inappropriate to omit affiliation, and in some (if not most) cases e-mail address served the purpose of denoting the affiliation. Well, all cases in Configurations are fine, because over time they were revised, revamped, rearranged, so they are grew far from original. So let's say names can be omitted from Configurations...

Reply:
Yes, it's not consistent.  I tried to look at each one and determine what to do.  That is, it was a manual edit, not an automatic one.